File: solutions-and-examples/java-projects/hello-java/src/Employee.java
public class Employee {
    private int id;
    private String name;
    private String email;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        if(id<=0){
            throw new RuntimeException("Id cannot be negative");
        }
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void printEmployee(){
        System.out.println("ID : "+id);
        System.out.println("Name : "+name);
        System.out.println("Email : "+email);
    }

}

File: angular-and-typescript/typescript-demos/Employee.js
var Employee = /** @class */ (function () {
    function Employee(id, name, email) {
        this.id = id;
        this.name = name;
        this.email = email;
    }
    return Employee;
}());
var employee1 = new Employee(101, 'Harsh', 'harsh@yahoo.com');
console.log(employee1);
